website/cue/reference/components/sinks/base/amqp.cue

```diff
@@ -57,6 +57,91 @@ base: components: sinks: amqp: configuration: {
 					type: string: examples: ["{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"message\", \"type\": \"string\" }] }"]
 				}
 			}
+			cef: {
+				description:   "The CEF Serializer Options."
+				relevant_when: "codec = \"cef\""
+				required:      true
+				type: object: options: {
+					device_event_class_id: {
+						description: """
+																Unique identifier for each event-type. Identifies the type of event reported.
+																The value length must be lower or equal to 1023.
```

Comment:
```suggestion
																The value length must be less than or equal to 1023.
```

Reply:
Fixes everywhere I managed to find this, thanks